openedx/core/djangoapps/content/block_structure/models.py

```diff
@@ -206,3 +264,18 @@ def _get_all_files(cls, data_usage_key):
             for filename in filenames
             if filename and not filename.startswith('.')
         ]
+
+    @classmethod
+    def _log(cls, bs_model, operation, serialized_data):
```

Comment:
**Note to reviewers:** I added a common helper for logging storage-access operations.  We now include the `storage.__class__`, `bucket_name` (used by S3 storage), and `location` (used by FS storage) in the log statements for easier debugging.  Would have been useful for debugging on Stage and may come in handy in the future.